pkg/virt-handler/vm.go

```diff
@@ -2561,6 +2564,72 @@ func (d *VirtualMachineController) vmUpdateHelperMigrationTarget(origVMI *v1.Vir
 	}
 	return nil
 }
+
+func ConfigureHousekeepingCgroup(vmi *v1.VirtualMachineInstance, d *VirtualMachineController) error {
+	cgroupManager, err := cgroup.NewManagerFromVM(vmi)
+	if err != nil {
+		return err
+	}
+
+	exists, err := cgroupManager.CreateChildCgroup("housekeeping", "cpuset")
+	// bail out if cgroup already configured
+	if exists == true {
+		return nil
+	}
+	if err != nil {
+		log.Log.Reason(err).Error("CreateChildCgroup ")
+		return err
+	}
+
+	key := controller.VirtualMachineInstanceKey(vmi)
+	domain, _, _, err := d.getDomainFromCache(key)
```

Comment:
Can you also check that the domain exists?

Reply:
If the domain does not exist the error would be `nil`.
I think we need to check the `exists` return value explicitly.

Reply:
Fixed.